### ISSUE TITLE
What's your methods to get Image and AnnoImage all be 384*384 ? 

### ISSUE THREAD

**Issue by AIML** [CSAILVision/sceneparsing#9]
Use the pre-trained model, I can't figure out the pixel-wise accuracy posted here. I wonder whether the resize on annotation image may heart the result

**Comment by hangzhaomit** [comment]
Resizing will affect the numbers you got slightly. We use "nearest" to resize annotation masks, and "bicubic" to resize images.

**Comment by AIML** [comment]
Thanks your reply very much! I use "nearest" to resize annotation masks, But the question still here. I use the evaluation code( I post it in the following ), and use your torch code load pre-trained model as opt.cnn, then run eval_utils.evalFunc. I only get pixel_acc: 0.522370。I beg your help!
function eval_utils.evalFunc(opt)                                                                                 
 10   if opt.gpu >= 0 then                                                                                            
 11     require 'cutorch'
 12     require 'cunn'
 13     cutorch.manualSeed(123)
 14     cutorch.setDevice(opt.gpu + 1)
 15   end
 16   
 17   local cnn = opt.cnn
 18   cnn:evaluate()                                                                                                  
 19   local crit = opt.crit
 20   
 21   local n = 0
 22   local loss_sum = 0
 23   local loss_evals = 0
 24   local pixel_correct = 0
 25   local pixel_labeled = 0 
 26   local predictions = {}                                                                                          
 27   
 28   -- Load pic Name                                                                                                
 29   local files = {}
 30   local aux_file = opt.aux_file 
 31   for line in io.lines(aux_file) do
 32     table.insert(files,line)                                                                                      
 33   end
 34   
 35   for _,name in pairs(files) do
 36     -- fetch a batch of data
 37     print(name)
 38     local img = image.load( opt.image_path..name..'.jpg', 3):mul(255):type(opt.dtype)                             
 39     img = img:view(1,3, opt.sample_size, opt.sample_size)
 40     local anno = image.load( opt.anno_path..name..'.png',1,'byte'):type(opt.dtype)                                
 41     anno:add(1)
 42     img, anno = net_utils.prepro( img, anno, opt.gpu >= 0) -- preprocess in place, and don't augment              
 43     n = n + 1                                                                                                     
 44     
 45     -- forward the model to get loss                                                                              
 46     local feats = cnn:forward(img)
 47     --[[
 48     local loss = crit:forward(feats, anno)
 49     loss_sum = loss_sum + loss
 50     --]]
 51     loss_evals = loss_evals + 1
 52     -- turn the prob mask into semantic segmentation mask
 53     local _, predMasks = torch.max(feats, 2)
 54     predMasks = torch.squeeze(predMasks)
 55     -- change size
 56     if predMasks:size(1)~=anno:size(2) or predMasks:size(2)~=anno:size(3) then                                    
 57       predMasks = image.scale(predMasks:view(1, predMasks:size(1),  predMasks:size(2)):type('torch.ByteTensor'), anno:size(2), anno:size(3), 'simple')
 58     end                                                                                                           
 59     predMasks = predMasks:byte()
 60     anno = anno:byte()                                                                                            
 61     -- calculate accuracy
 62     local _, cur_correct, cur_labeled = utils.pixelAccuracy(predMasks-1, anno-1)
 63     pixel_correct = pixel_correct + cur_correct
 64     pixel_labeled = pixel_labeled + cur_labeled
 65     print(string.format('evaluating validation performance: loss:%f   pixel_acc:%f ',                             
 66                            loss_sum/loss_evals, pixel_correct/pixel_labeled))                                     
 67     if loss_evals % 10 == 0 then collectgarbage() end
 68   end
 69   collectgarbage() 
 70   return loss_sum/loss_evals, pixel_correct/pixel_labeled, predictions                                            
 71 end

**Comment by hangzhaomit** [comment]
@AIML Hi I figured out the bug recently, and just updated script to load the caffe model. What I missed before is to copy bias to the torch model.
